set_retry_4_times.yml

```diff
@@ -0,0 +1,19 @@
+---
+# This playbook is used to test the retry logic in a CloudForms state machine
```

Comment:
MIQ instead of CloudForms here please :)

Reply:
I have forked Devel repository, but I have updated it.